### ISSUE TITLE
Images incorrectly shared from Gallery

### ISSUE THREAD

**Issue by SlavikCA** [nextcloud/gallery#343]
### Steps to reproduce

1. Using NC 13 beta 3
2. Open NextCloud in the browser
3. Click on any image. Image displayed, as expected
4. Click on Share button 
5. Check "Share link"
6. Open that link (example of the link, I get: https://nc.slavikf.com/index.php/apps/gallery/s/x4xXjb9BXif7Dy8)

### Expected behaviour

Image opened in the browser - same way, as it shown to me in the gallery. 

Note, that if Share from "Files" (not from "Gallery) - it shared correctly: I'm getting link like this: https://nc.slavikf.com/index.php/s/x4xXjb9BXif7Dy8 and image is shown in the browser, as expected.

### Actual behaviour

Full-size Image (~6MB in the example above) downloaded.
So, Share via Link effectively is Download, which is not expected.

### Server configuration

- Ubuntu 16.04
- Apache web server
- MySQL DB
- PHP 7.0
- NextCloud 13 beta 3
- Upgraded from NC 12

**Are you using external storage, if yes which one:** local folder, smb share, sftp, etc.

No

**Are you using encryption:** yes/no

No

**Are you using custom gallery.cnf config files:** yes/no

No

### Client configuration

**Browser:**

Chrome 63 on Mac OS 10.12.6

**Comment by SlavikCA** [comment]
Probably related to https://github.com/nextcloud/server/issues/7614

**Comment by Zuikkis** [comment]
I wonder why this is still not fixed, bug is present in current version and easy to reproduce.

The problem is in the http response headers. Gallery headers have:

Content-Disposition: attachment;

that should be:

Content-Disposition: inline;

This is easily "fixed" in the file "apps/gallery/lib/Http/ImageResponse.php", near line 36.. Old line commented out and new line below:

```
        public function __construct(array $image, $statusCode = Http::STATUS_OK) {
                $name = $image['name'];
                $this->preview = $image['preview'];

                $this->setStatus($statusCode);
                $this->addHeader('Content-type', $image['mimetype'] . '; charset=utf-8');
                $this->addHeader('Content-Disposition',
//                                               'attachment; filename*=UTF-8\'\'' . rawurlencode($name) . '; filename="'
                                                 'inline; filename*=UTF-8\'\'' . rawurlencode($name) . '; filename="'
                                                 . rawurlencode($name) . '"'
                );
        }

```

However, this has the side effect that "download" button no longer downloads but instead opens the picture in browser. But it's easy to then right click and "save as" there...  

But perhaps someone can make a better fix. :)

**Comment by skjnldsv** [comment]
Fixed by #90
